astro.config.mjs

```diff
@@ -11,8 +15,20 @@ export default defineConfig({
   },
   site: "https://docs.solvro.pl",
   integrations: [
+    liveCode({
+      defaultProps: {
+        "client:load": true,
+      },
+    }),
     starlight({
       defaultLocale: "pl",
+      locales: {
+        root: {
+          label: "Polski",
+          lang: "pl",
+        },
+      },
+      favicon: "favicon.ico",
```

Comment:
będziemy robić multi-lang?

Reply:
nope, to jest tylko po to, żeby wymusić język polski bo u mnie po angielsku było